mmv1/products/oracledatabase/CloudVmCluster.yaml

```diff
@@ -66,10 +72,21 @@ examples:
       project: 'my-project'
       cloud_vm_cluster_id: 'my-instance'
       cloud_exadata_infrastructure_id: 'my-exadata'
+      deletion_protection: 'true'
+    ignore_read_extra:
+      - 'deletion_protection'
     test_vars_overrides:
+      'deletion_protection': 'false'
       'project': '"oci-terraform-testing"'
       'cloud_vm_cluster_id': '"ofake-vmcluster-full"'
       'cloud_exadata_infrastructure_id': '"ofake-exadata-for-vm-full"'
+virtual_fields:
+  - name: 'deletion_protection'
```

Comment:
What about using camel case here?

Reply:
I think this is recommend format https://googlecloudplatform.github.io/magic-modules/develop/client-side-fields/

Reply:
Ah I see, thanks for checking out.